internal/locate/region_request.go

```diff
@@ -186,20 +234,75 @@ func (r *RegionRequestRuntimeStats) Merge(rs RegionRequestRuntimeStats) {
 		stat.Count += v.Count
 		stat.Consume += v.Consume
 	}
+	if len(rs.ErrStats) > 0 {
+		if r.ErrStats == nil {
+			r.ErrStats = make(map[string]int)
+		}
+		for err, cnt := range rs.ErrStats {
+			r.ErrStats[err] += cnt
+		}
+	}
 }
 
-// RecordRegionRequestRuntimeStats records request runtime stats.
-func RecordRegionRequestRuntimeStats(stats map[tikvrpc.CmdType]*RPCRuntimeStats, cmd tikvrpc.CmdType, d time.Duration) {
-	stat, ok := stats[cmd]
-	if !ok {
-		stats[cmd] = &RPCRuntimeStats{
-			Count:   1,
-			Consume: int64(d),
-		}
-		return
+// ReplicaAccessStats records the first 20 access info, after more than 20 records, count them by `OverflowCount` field.
+type ReplicaAccessStats struct {
+	AccessInfos   []ReplicaAccessInfo
+	OverflowCount int
```

Comment:
How about merging the access infor by peer when there are many access following? `Overflowcount` could not provide useful information.

Reply:
Since `ReplicaAccessStats ` currently only records for this round, and in normal case, each replica should only access only once, for leader replica can be access 10 times, so normally, OverflowCount should always be 0.

Reply:
What about the `follower read` or `stale read` cases? The `OverflowCount` still does not seem very helpful.
Consider such a case, the `NotLeader` error is encountered serveral times because of region election, after the new leader information is returned to the kv-client, some new errors caused by disk io problems are encountered, while the related information is recorded as `OverflowCount` .

Reply:
great, done. PTAL